kivy/uix/filechooser.py

```diff
@@ -106,7 +106,10 @@
     basename, join, sep, normpath, expanduser, altsep,
     splitdrive, realpath, getsize, isdir, abspath, isfile, dirname)
 from fnmatch import fnmatch
-import collections
+try:
+    import collections.abc as collections_abc
+except ImportError:
+    import collections as collections_abc
```

Comment:
> Deprecated since version 3.3, will be removed in version 3.9: Moved Collections Abstract Base Classes to the collections.abc module. For backwards compatibility, they continue to be visible in this module through Python 3.8."

I think that should be safe for us to just change it to `import collections.abc` and then use `is instance(file, collections.abc.Callable)`, cause we don't mark `kivy` as supported on `Python<=3.3`.